packages/design-system/src/components/icon-button-with-menu.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import { CheckIcon, IconComponent } from "@webstudio-is/icons";
+import * as DropdownMenuPrimitive from "@radix-ui/react-dropdown-menu";
+import { styled } from "../stitches.config";
+import { Tooltip } from "./tooltip";
+
+const StyledContent = styled(DropdownMenuPrimitive.Content, {
+  maxWidth: "$sizes$dropdownMenu",
+  backgroundColor: "$colors$slate4",
+  borderRadius: "$radii$1",
+  padding: "$sizes$2",
+  boxShadow:
+    "0px 2px 7px rgba(0, 0, 0, 0.1), 0px 5px 17px rgba(0, 0, 0, 0.15), inset 0 0 1px 1px $colors$gray1, 0 0 0 1px $colors$gray8",
+});
+
+const itemStyles = {
+  all: "unset",
+  fontSize: "$2",
+  lineHeight: 1,
+  color: "$colors$slate12",
+  borderRadius: "$radii$1",
+  display: "flex",
+  alignItems: "center",
+  height: "$sizes$5",
```

Comment:
why $sizes$5 and not $5? just wondering maybe you don't know or maybe I don't know why this is better?

Reply:
ping